tensorflow/contrib/rnn/BUILD

```diff
@@ -8,12 +8,37 @@ exports_files(["LICENSE"])
 
 package(default_visibility = ["//tensorflow:__subpackages__"])
 
+load("//tensorflow:tensorflow.bzl", "tf_custom_op_library")
+load("//tensorflow:tensorflow.bzl", "tf_gen_op_libs")
+load("//tensorflow:tensorflow.bzl", "tf_gen_op_wrapper_py")
 load("//tensorflow:tensorflow.bzl", "cuda_py_tests")
 
+tf_custom_op_library(
+    name = "python/ops/_lstm_ops.so",
+    srcs = [
+        "ops/lstm_ops.cc",
+    ],
+    hdrs = ["kernels/lstm_ops.h"],
+    gpu_srcs = [
+        "kernels/lstm_ops.cc",
```

Comment:
@keveman is this right?  do CPU srcs also go in gpu_srcs?  I think youll end up using nvcc to compile your kernel, which may not be as fast.